client/Cargo.toml

```diff
@@ -28,8 +28,8 @@ pallet-encointer-communities = { git = "https://github.com/encointer/pallets", b
 pallet-encointer-scheduler = { git = "https://github.com/encointer/pallets", branch = "master" }
 
 # scs deps
-substrate-api-client = { git = "https://github.com/scs/substrate-api-client", branch = "master"}
-substrate-client-keystore = { git = "https://github.com/scs/substrate-api-client", branch = "master"}
+substrate-api-client = { git = "https://github.com/scs/substrate-api-client", branch = "cl/encointer-community-currency-fees-hotfix"}
```

Comment:
not sure I'd like to merge that yet with this branch reference. maybe wait for a proper solution?